src/media.cls.php

```diff
@@ -724,6 +725,20 @@ private function _parse_img_for_preload() {
 			Debug2::debug2('[Media] VPI preload found and matched: ' . $attrs['src']);
 
 			$this->_vpi_preload_list[] = $attrs['src'];
+
+			// Add attributes fetchpriority="high" and decode="sync"
+			// after WP 6.3.0 use: wp_img_tag_add_loading_optimization_attrs().
+			$new_html = [];
+			$attrs[ 'fetchpriority' ] = 'high';
+			$attrs[ 'decoding' ] = 'sync';
+			// create html with new attributes.
+			foreach( $attrs as $k => $attr ) {
+				$new_html[] = $k . '="' . $attr . '"';
+			}
+
+			if( $new_html ) {
+				$this->content = str_replace( $match[1], implode( ' ', $new_html), $this->content );
```

Comment:
@timotei-litespeed please be aware that, replacement in loop could be low efficient. Should replace outside of loop once only.